### PR TITLE
015 release

### PR DESCRIPTION
**Pull request by b264**
Removes Ruby 1.9.2.  Randomizes authors.  Bumps version to 1.6.0

### CODE REVIEW

**Comment by NikitaAvvakumov** [comment]
Considering that all 1.9 is unsupported as of February, should we drop 1.9.3 in favour of >= 2.0.0?

**Comment by b264** [comment]
I would prefer to support 1.9.3 unless it really buys us something not to.

File: factory-helper.gemspec [hunk 1]
```diff
@@ -6,13 +6,13 @@ require 'factory-helper/version'
 Gem::Specification.new do |spec|
   spec.name          = "factory-helper"
   spec.version       = FactoryHelper::VERSION
-  spec.authors       = ["Brian Giaraffa", "Nikita Avvakumov"]
+  spec.authors       = ['Brian Giaraffa', 'Nikita Avvakumov'].permutation.to_a.sample
```
Comment (NikitaAvvakumov) [inline factory-helper.gemspec:9]: Nice

Comment (b264) [inline factory-helper.gemspec:9]: :-D

**Comment by NikitaAvvakumov** [comment]
:shipit:

**Pull request merged**